dev-environment.yml

```diff
@@ -26,3 +26,4 @@ dependencies:
   - sphinx-gallery
   - scikit-image
   - pyyaml
+  - rioxarray
```

Comment:
Why is it needed now?

Reply:
For typing in `raster.py`: https://github.com/GlacioHack/geoutils/blob/main/geoutils/georaster/raster.py#L1756

This was caught by `sphinx-autodoc-typehints` that raised an error during the sphinx build (and thus made our tests fail). This error was new, and the cause seemed to have been the upgrade of `sphinx-autodoc-typehints`.

Adding this to the description.